install_helper.sh

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
+set -e
+
+get_releases() {
+  curl --silent "https://api.github.com/repos/cdr/code-server/releases/latest" |
+    grep '"browser_download_url":\|"tag_name":'
+}
+
+linux_install() {
+  bin_path=$HOME/bin
```

Comment:
We shouldn't be installing it in ~/bin. It should be going into `/usr/local/bin`. Or just the current directory.

Reply:
I was thinking it'd be nice to not require `sudo` perms... although requiring `sudo` seems better than installing everything into the current dir

Reply:
We can also use /opt which does not require sudo iirc.

Reply:
Another option is to create a new hidden dir `~/.code-server` and put everything there. Then we could either tell the user to add it to their path or just do it ourselves?

Reply:
Lets do whatever fzf does.